unicodetools/data/ucd/dev/DoNotEmit.txt

```diff
@@ -42,6 +42,14 @@
 # Sequences for Egyptian Hieroglyphs are not listed here. See
 # the kEH_AltSeq property in UAX #57 for that information.
 #
+# CJK compatibility ideographs are not listed here either. Each CJK
+# compatibility ideograph is canonically equivalent to a CJK unified
+# ideograph, which means that distinctions would be lost in normalization.
```

Comment:
Should we say something about the 12 CJK compatibility ideographs which do not have a decomposition?

NamesList.txt:
```
@		The IBM 32 compatibility ideographs
@+		This section contains twelve characters that are CJK unified ideographs. Each CJK unified ideograph is annotated as such.
FA0E	CJK COMPATIBILITY IDEOGRAPH-FA0E
	* a CJK unified ideograph
...
```

[[:Block=CJK_Compatibility_Ideographs:]&[:Unified_Ideograph:]](https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5B%3ABlock%3DCJK_Compatibility_Ideographs%3A%5D%26%5B%3AUnified_Ideograph%3A%5D&g=&i=)

@kenlunde FYI

Reply:
> Should we say something about the 12 CJK compatibility ideographs which do not have a decomposition?

I had forgotten about those! I modified the language to cover those, as well as be more explicit about the whole idea. PTAL.